src/components/OnboardingLoader.js

```diff
@@ -0,0 +1,28 @@
+import React, { useCallback, useState } from 'react'
+import OnboardingModal from './Onboarding/OnboardingModal'
+
+function OnboardingLoader({ children }) {
+  const [onboardingCompleted, setOnboardingCompleted] = useState(
+    localStorage.getItem('onboardingCompleted')
+  )
+
+  const handleStartCourt = useCallback(() => {
+    localStorage.setItem('onboardingCompleted', true)
+    setOnboardingCompleted(true)
+  }, [])
+
+  return (
+    <React.Fragment>
+      {!onboardingCompleted && (
+        <OnboardingModal
+          visible={!onboardingCompleted}
```

Comment:
Did we decide to show the modal immediately on start up rather than after connecting the account?

Reply:
For now it’s on start up, but if we decide to show it after the connection, then I think we should have the providers / connection workflow into the onboarding modal itself. Otherwise I think it would be a bit disturbing to move from a new state (the connected account module) to another one immediately (the modal opening).

Reply:
Now that we're not asking for email addresses, let's keep it at start up.